nixd/tools/nixd/test/semantic-tokens/basic.md

```diff
@@ -33,24 +32,8 @@
     string2 = "${builtins.foo}";
     undefined = x;
     list = [ 1 2 3 ];
-    null = null;
-    wit = with builtins; [ a b c];
```

Comment:
An example of how the duplicated nix block didn't match the actual document text.